cookbook/web-server/nginx.rst

```diff
@@ -60,5 +60,23 @@ The Nginx configuration could look something like.
 .. warning::
     Be sure to also configure your local host-file, if running Sulu locally.
 
-.. include:: file-permissions.inc.rst
+File upload
+-----------
+
+By default the nginx has a file limit of 2M when uploading files.
```

Comment:
"By default nginx has a file limit of 2MB"

Reply:
fixed